subsets/Makefile

```diff
@@ -13,10 +13,10 @@ taxslim: taxon-subset-ids.txt
 	robot convert -i taxslim.owl -f json -o taxslim.json
 
 taxslim-disjoint-over-in-taxon.owl: taxslim
-	owltools taxslim.owl --create-taxon-disjoint-over-in-taxon --root NCBITaxon:1 --output $@.tmp.owl
+	owltools taxslim.owl --create-taxon-disjoint-over-in-taxon --root NCBITaxon:1
```

Comment:
How is the output of this command written?

Reply:
`owltools` magic 🪄

Reply:
but it's the same way as the before

Reply:
if (output==null) o.write(input)?

Reply:
This is highly dependent on the parameter `--create-taxon-disjoint-over-in-taxon`. I cannot answer more than that 😬

Reply:
Well, I hope @balhoff will know how this works, I don't :P

Reply:
I don't either!